changes/2910.bugfix

```diff
@@ -0,0 +1 @@
+fix cancellation broadcast during DNS resolve
```

Comment:
I'm sorry but the folder name must be upper-cased: `CHANGES/2910.bugfix`.
The idea is having change-note on the top of changed files list.

Reply:
I think this folder was renamed in master, so I ran into this: https://coderwall.com/p/5w3jca/git-case-sensitivity-on-os-x   fixed